### ISSUE TITLE
feat(reddit): Color comment indents

### ISSUE THREAD

**Issue by jordan1776** [ReVanced/revanced-patches#742]
### Application

reddit

### Issue

This is an addendum to the request ReVanced/revanced-patches-template#787. It can be hard to follow threads with many replies. The current Official reddit app shows white bars for replies with comments.

[This is how it appears.](https://i.imgur.com/qOT4ySM.png)

If ReVanced/revanced-patches-template#787 were to be implemented, implementing colored replies as an option would further increase readability.

### Patch

The official reddit app should support colored comment indents like Sync for reddit.

For example: [Like this](https://i.imgur.com/kXCuZd6.png)

### Motivation

This would make following threads with many layers of replies much easier.

### Acknowledgements

- [X] I have searched the existing issues and this is a new and no duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I filled out all of the requested information in this issue properly.

**Comment by oSumAtrIX** [comment]
Over time, a lot of feature requests have been made, which is why I am closing some according to the [contribution guidelines](https://github.com/ReVanced/revanced-patches/blob/main/CONTRIBUTING.md#-submitting-a-feature-request):

> Requests can be accepted or rejected at the discretion of maintainers of ReVanced Patches

This feature request has been closed for one or more of the following reasons:

- Stale for too long
- Low priority
- Maintainers un-interested
- Too complex to realize

Regardless, anyone who comes across this feature request can PR it. Thanks for the feature request!
